modules/macros/pages/link-macro-attribute-parsing.adoc

```diff
@@ -22,26 +22,31 @@ The following example shows a URL macro with custom linked text alongside named
 https://discuss.asciidoctor.org[Discuss Asciidoctor,role=resource,window=_blank]
 ----
 
-If you want to define a single named attribute, but leave the linked text empty (so it falls back to the URL), then you need to start the attribute list with a comma.
-In other words, you need to leave a slot for the linked text so that the parser knows where it ends.
-Otherwise, the parser will treat the named attribute as the linked text.
+Let's consider a case where the linked text contains a comma and the macro also has named attributes.
+In this case, you must enclose the linked text in double quotes so that it is recognized as a single (positional) attribute.
 
 [source]
 ----
-https://discuss.asciidoctor.org[,role=resource]
+https://example.org["Google, Yahoo, Bing",role=teal]
 ----
 
-When attribute parsing is enabled, the link macro recognizes all the common attributes (id, role, and opts).
-It also recognizes a handful of attributes that are specific to the link macro.
+Similarly, if the linked text contains an equals sign, the text must be enclosed in double quotes.
 
-Let's consider a case when the linked text contains a comma and macro also has named attributes.
-In this case, you must enclose the linked text in double quotes so that it gets preserved by the parser.
+[source]
+----
+https://example.org["The problem of inequality: does 1=2?"]
```

Comment:
This is not actually a valid use case. The equal sign must occur before the first space. Otherwise, what precedes it is not considered to be an attribute name and thus not parsed as an attribute.

You could say "1=2 posits the problem of inequality"

Reply:
Indeed, thanks!